spec/lib/reports/orders_and_fulfillment/orders_cycle_supplier_totals_report_spec.rb

```diff
@@ -67,7 +67,8 @@
 
     it "is summarised" do
       expect(report).to receive(:display_summary_row?).and_return(true)
-
+      # assures product appears first on report table
+      variant.product.update!(name: "Alpha-Product #000")
       variant.product.update!(variant_unit: "weight")
```

Comment:
Doing both in one `update!` call would avoid an unnecessary query and save a tiny bit of time.